draft-ietf-jsonpath-base.md

```diff
@@ -1474,13 +1474,23 @@ Queries:
 
 ## Normalized Paths
 
-A Normalized Path is a JSONPath with restricted syntax that identifies a node by providing a query that results in exactly that node. For example,
-the JSONPath expression `$.book[?(@.price<10)]` could select two values with Normalized Paths
-`$['book'][3]` and `$['book'][5]`. For a given JSON value, there is a one to one correspondence between the value's
-nodes and the Normalized Paths that identify these nodes.
-Note that there is precisely one Normalized Path that identifies each node.
+A Normalized Path is a JSONPath with restricted syntax that identifies a node by providing a query that results in exactly that node,
+for example, `$['book'][3]`.
+
+A Normalized Path provides a concrete string representation of a node in a nodelist.
```

Comment:
An address is not a representation.

Reply:
We may want to be explicit that these paths are a **convention** for expressing **node identify**.
(This is part of the specification because identifying a "conventional" convention is good, as the text below says.)

Reply:
Hopefully this is now covered.